cmd/payledger/bar.go

```diff
@@ -11,6 +11,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/Qitmeer/qitmeer/common/roughtime"
 	"os"
 	"time"
```

Comment:
1.) The `roughtime` and `time` package are both imported,  what's the purpose on the case? or is it just a mistake that forgetting to remove the `time` package ?
2.) There are several files contains the same usage in 1.).  I think it may be better to rename `time` import by using syntax like
```go
 import another_name "package"
``` 
Th syntax can eliminate the import of both package, if the case in 1.) is do a mistake,  the `roughtime` should replace all the palace where `time` existed on originally. otherwise, I might wonder when should use official `time` package intentionally？

Reply:
Because `roughtime` only modifies a small part of the interface, other functions still need system `time`.